### PR TITLE
Changing registry image to fix CI failure

### PR DESCRIPTION
**Pull request by varshab1210**
**What type of PR is this?**

/kind failing-test

**What does this PR do / why we need it**:
Fixes CI failure

**Have you updated the necessary documentation?**

* [ ] Documentation update is required by this PR.
* [ ] Documentation has been updated.

**Which issue(s) this PR fixes**:

Fixes #?
https://issues.redhat.com/browse/GITOPS-4993

**Test acceptance criteria**:

* [ ] Unit Test
* [ ] E2E Test

**How to test changes / Special notes to the reviewer**:

### CODE REVIEW

File: openshift-ci/build-root/Dockerfile [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # Dockerfile to bootstrap build and test in openshift-ci
 
-FROM registry.ci.openshift.org/openshift/release:golang-1.19
+FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.21-openshift-4.16
 
 ARG KUBECTL_KUTTL_VERSION=0.12.1
 ARG OPERATOR_SDK_VERSION=1.22.2
 
-RUN yum -y install dnf httpd-tools podman docker
+RUN dnf -y install dnf httpd-tools podman docker
```
Comment (deepsm007) [inline openshift-ci/build-root/Dockerfile:8]:
```suggestion
RUN dnf -y install httpd-tools podman docker
```
Or
 
```suggestion
RUN yum -y install httpd-tools podman docker
```

Comment (varshab1210) [reply openshift-ci/build-root/Dockerfile]: Thanks @deepsm007, addressed

```
/bin/sh: dnf: command not found
```

But locally this does pass, not sure what's happening

Comment (deepsm007) [reply openshift-ci/build-root/Dockerfile]: try with `yum -y install httpd-tools podman docker `

Comment (varshab1210) [reply openshift-ci/build-root/Dockerfile]: Still failing

Comment (varshab1210) [reply openshift-ci/build-root/Dockerfile]: For now, I am trying this without installing httpd-tools podman and docker. If its not required by our tests, I will remove them. But if that's the case, why do we need to update the image? This issue was occurring even when the image was not changed

**Review by anandf — APPROVED** [review]

**Pull request merged**